README.md

```diff
@@ -11,6 +11,7 @@ A re-introduction of the web application for ESRG Knights of the Kitchen Table.
 1. Install the dependencies: `pip install -r requirements/dev.txt`. These dependencies include common dependencies (such as *Django*) as well as dev-dependencies that speed up or ease the development process (such as *coverage.py*). For more information about dependencies, view the *Dependencies* section below.
 1. Setup the database by running `python manage.py migrate`. This ensures your database can store the items we expect to store in it.
 1. Start the server: `python manage.py runserver`. This starts a web server, which you can access using your webbrowser and going to `localhost:8000`.
+1. If wanting to use functionality that involves sending emails (such as resetting a password), then you'll also need to run the following command in another command prompt/terminal: `python -m smtpd -n -c DebuggingServer localhost:1025`. This will mimic an smtp email server. Any emails that would normally be sent will instead show up in this terminal.
```

Comment:
I do not know if ```python -m smtpd -n -c DebuggingServer localhost:1025``` is correct

Reply:
It was taken straight from the [documentation](https://docs.djangoproject.com/en/2.2/topics/email/#configuring-email-for-development).